### PR TITLE
Origin of xeno job point recorded

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Tally the origin of all larva point in the round and show it with the rest of the statistic at the end of the round

## Why It's Good For The Game

I still think their is too much xenos compared to the amount of marines. This will help to balance it, cause I don't know if it's silo producing too much, marine spawning giving too much or psy drain

## Changelog
:cl:
code: Show the origin of larva points in statistic
/:cl:

### CODE REVIEW

File: code/datums/jobs/job/xenomorph.dm [hunk 1]
```diff
@@ -11,6 +11,10 @@
 /datum/job/xenomorph/return_spawn_type(datum/preferences/prefs)
 	return /mob/living/carbon/xenomorph/larva
 
+/datum/job/xenomorph/add_job_points(amount, origin = MARINE_SPAWN_ORIGIN)
+	. = ..(amount)
+	SSblackbox.record_feedback("tally", "round_statistics", 0.125, origin)
```
Comment (SplinterGP) [inline code/datums/jobs/job/xenomorph.dm:16]: shoulnd't it  be amount instead of 0.125 ?

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/datums/jobs/job/xenomorph.dm (hunk 1 above)
Comment (BraveMole) [reply code/datums/jobs/job/xenomorph.dm:16]: should be amount * 0.125, me stoopid

Comment (BraveMole) [inline code/datums/jobs/job/xenomorph.dm:16]:
```suggestion
	SSblackbox.record_feedback("tally", "round_statistics", 0.125 * amount, origin)
```

File: code/datums/jobs/job/xenomorph.dm [hunk 2]
```diff
@@ -11,6 +11,10 @@
 /datum/job/xenomorph/return_spawn_type(datum/preferences/prefs)
 	return /mob/living/carbon/xenomorph/larva
 
+/datum/job/xenomorph/add_job_points(amount, origin = MARINE_SPAWN_ORIGIN)
+	. = ..(amount)
```
Comment (SplinterGP) [inline code/datums/jobs/job/xenomorph.dm:15]: don't believe this is needed
```suggestion
	. = ..()
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**